client/influxdb_test.go

```diff
@@ -517,8 +517,8 @@ func TestClient_Timeout(t *testing.T) {
 	_, err = c.Query(query)
 	if err == nil {
 		t.Fatalf("unexpected success.  expected timeout error")
-	} else if !strings.Contains(err.Error(), "use of closed network connection") {
-		t.Fatalf("unexpected error.  expected 'use of closed network connection' error, got %v", err)
+	} else if !strings.Contains(err.Error(), "request canceled") {
```

Comment:
@corylanou -- please confirm that this is OK with you. I think the error must have changed in Go 1.5.